scripts/test-e2e-local.js

```diff
@@ -34,7 +34,7 @@ const {
 const {
   downloadHermesSourceTarball,
   expandHermesSourceTarball,
-} = require('./hermes/hermes-utils');
+} = require('../packages/react-native/scripts/hermes/hermes-utils.js');
```

Comment:
I wonder if we could use `react-native/scripts/hermes/hermes-utils.js` here, since we are monorepo now :)

I am expecting that `node_modules/react-native` should be a symlink now and this might work. Could you please test this?

Reply:
I'll try it out and report back 👍

Reply:
yup, it's working! thanks Ruslan 👍